### PR TITLE
Add 4 new magazines focused on flintlocks

### PR DESCRIPTION
**Pull request by jokermatt999**
This update adds a new magazine, Muzzleloaders Quarterly. It's got 4 editions in game, moderately rare in spawns currently. They function as low level gun magazines but with the recipes for flintlock pistol, flintlock rifle, flintlock shots and black powder recipe, so worth keeping around. Shouldn't be too OP, but may nudge players towards a fairly underused weapon. Also, Cataclysm could use some more patriotic flavor this July 4th weekend!

*Yeah, it's been a while since I contributed (if you remember me). I might add some json now and then, but I likely will leave the coding to better programmers.

### CODE REVIEW

**Comment by mugling** [comment]
I like the idea but I'm not sure if this shouldn't be a single magazine?

**Comment by jokermatt999** [comment]
I also wanted to have a "collect them all!" aspect.

**Comment by mugling** [comment]
The other magazines aren't like that and I don't think they're particularly useful containing one recipe only

**Comment by Coolthulhu** [comment]
Yeah, having just one cover all of that would be much better.
Collectibles with low value and similar names are the worst.

**Comment by kevingranade** [comment]
I'm on the fence, I think I'm leaning toward mugling's opinion on this one, but I can't decide if that's because of the effect on players or because of all the boilerplate for the definitions.

BTW mugling, books/magazines are probably one of the clearest wins for inheritance, there are literally 5 classes of reading material, the members of which are only separated by id, name, description and a handful of book-specific stats.

**Comment by jokermatt999** [comment]
So I consolidated it into one issue. It's now Guns 0-2, has recipes for flintlock guns at a lower level, and contains a difficult black powder recipe (6, as the AAA guide.)

File: data/json/items/books.json [hunk 1]
```diff
@@ -898,6 +898,26 @@
     },
     {
         "type" : "BOOK",
+        "id" : "mag_muzzle",
+        "name" : "Muzzle Loaders Annual",
+        "name_plural" : "Muzzle Loaders Annual",
+        "max_level" : 2,
+        "description" : "A hefty annual publication on traditional muzzleloading guns. It covers the design, building, and care of several flintlock weapons with tools that can be found at home. There's also a well written, albeit technical article on the basic chemistry of black powder. Plus it comes with a free Redcoat silhouette shooting target!",
```
Comment (mugling) [inline data/json/items/books.json]: Double spacing after stops

File: data/json/itemgroups/books.json [hunk 2]
```diff
@@ -23,7 +24,8 @@
       [ "manual_shotgun", 30 ],
       [ "manual_smg", 10 ],
       [ "pocket_firearms", 100 ],
-      [ "recipe_bullets", 80 ]
+      [ "recipe_bullets", 80 ],
+      [ "mag_muzzle", 30 ]
```
Comment (mugling) [inline data/json/itemgroups/books.json]: Not here. This is for larger reference books

File: data/json/recipes/recipe_ammo.json [hunk 3]
```diff
@@ -2439,7 +2439,7 @@
   "time": 500,
   "reversible": false,
   "autolearn": true,
-  "book_learn": [["recipe_bullets" , 2 ] , [ "manual_shotgun", 2 ]],
+  "book_learn": [["recipe_bullets" , 2 ] , [ "manual_shotgun", 2 ] , [ "mag_muzzle", 2 ]],
```
Comment (mugling) [inline data/json/recipes/recipe_ammo.json]: Should be `mag_muzzle 1` considering the magazine is explicitly about that subject

File: data/json/recipes/recipe_weapon.json [hunk 4]
```diff
@@ -5686,7 +5687,7 @@
     "time": 240000,
     "reversible": false,
     "autolearn": true,
-    "book_learn": [[ "manual_rifle", 5 ] , [ "mag_rifle", 6 ]],
+    "book_learn": [[ "manual_rifle", 5 ] , [ "mag_rifle", 6 ] , [ "mag_muzzle", 5 ]],
```
Comment (mugling) [inline data/json/recipes/recipe_weapon.json]: `mag_muzzle 4`

File: data/json/recipes/recipe_medsandchemicals.json [hunk 5]
```diff
@@ -2620,7 +2620,7 @@
   "time": 20000,
   "reversible": false,
   "autolearn": false,
-  "book_learn": [[ "textbook_anarch", 6 ] , [ "recipe_labchem", 4 ], ["textbook_chemistry", 5], ["textbook_armschina", 5]],
+  "book_learn": [[ "textbook_anarch", 6 ] , [ "recipe_labchem", 4 ], [ "textbook_chemistry", 5 ] , [ "textbook_armschina", 5 ] , [ "mag_muzzle", 6 ]],
```
Comment (mugling) [inline data/json/recipes/recipe_medsandchemicals.json]: `mag_muzzle 3`

**Comment by pisskop** [comment]
why not make it a pirate themed book?  or civilwar enthusiast book?

out of cutiousity.  Youve followed the convention of the books present.


Like:

  -B_arrrg_in guns
  -Blunderbuss and booty
  -Authentic Replica Catalougue 2040
  -_My First Gun_
  -Privateer's Pistols

**Comment by mugling** [comment]
Nothing inherently wrong with this but appears abandoned

**Pull request closed without merging**